vnext/IntegrationTests/React.Windows.IntegrationTests.vcxproj

```diff
@@ -29,7 +29,6 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{700A84FD-F92A-43F1-8D06-B0E0745DF9B5}</ProjectGuid>
     <Keyword>StaticLibrary</Keyword>
-    <AppContainerApplication>true</AppContainerApplication>
```

Comment:
Why is this change needed? (Not that I mind).

Reply:
Without it, the office compiler ends up including vccorlib.h and then trips over the C++/CX language features in it, resulting in several errors like 

d:\iss\private-rnw\vnext\packages\visualcpptools.internaladdchpe.vs2017layout.14.13.26133.2\lib\native\include\vccorlib.h(934): error C2059: syntax error: ':' (compiling source file TestRootView.cpp) [D:\ISS\private-rnw\vnext\IntegrationTests\React.Windows.IntegrationTests.vcxproj]